roles/openssl_cert_gen/templates/xdmod.conf

```diff
@@ -0,0 +1,50 @@
+<IfModule !mod_ssl.c>
+    LoadModule ssl_module modules/mod_ssl.so
+</IfModule>
+
+Listen 443
```

Comment:
This line can be removed. Let ssl.conf handle it

Reply:
removed